tests/parser/exceptions/test_syntax_exception.py

```diff
@@ -35,7 +35,7 @@ def foo():
     z = x[2:4]
     """,
     """
-Transfer: event({_&rom: indexed(address)})
+Transfer: event({_rom&: indexed(address)})
```

Comment:
What is this change about?

Reply:
now that `&` is a valid operator, `_&rom` is actually valid syntax (`BITAND _ rom`). this changes the test so that it `BITAND _rom` fails as it is not a unary operator.